bundles/io/org.eclipse.smarthome.io.rest/src/main/java/org/eclipse/smarthome/io/rest/internal/util/Constants.java

```diff
@@ -0,0 +1,11 @@
+package org.eclipse.smarthome.io.rest.internal.util;
```

Comment:
I'd change the package to org.eclipse.smarthome.io.rest.internal - no need for a new subpackage for some constants.